QuestionAnswering/QA_ICL/data_generation/README.md

```diff
@@ -1,90 +1,56 @@
-# Data generation for QA applications using in-context learening
+# Data Generation for QA_ICL
 
-## Introduction
+## Table of Contents
 
-This directory contains scripts to generate three kinds of data: lexicon, simplified graph schema, and examples of data request generation.
+- [Data Generation for QA\_ICL](#data-generation-for-qa_icl)
+  - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
+  - [Terminologies](#terminologies)
+    - [Lexicon](#lexicon)
+    - [Schema Elements](#schema-elements)
+    - [Data Request Generation Examples](#data-request-generation-examples)
+    - [Physical Quantity Recognition Examples](#physical-quantity-recognition-examples)
+  - [Installation](#installation)
+    - [Prerequisites](#prerequisites)
+    - [Steps](#steps)
+  - [Usage](#usage)
+    - [Lexicon Generation](#lexicon-generation)
+    - [KG Schema Extraction](#kg-schema-extraction)
+    - [CSV-to-JSON Conversion of Data Request Generation Examples](#csv-to-json-conversion-of-data-request-generation-examples)
+    - [Augmentation of Data Request Generation](#augmentation-of-data-request-generation)
+    - [CSV-to-JSON Conversion of Physical Quantity Detection Examples](#csv-to-json-conversion-of-physical-quantity-detection-examples)
+  
+## Overview
 
```

Comment:
After reading the comments, I have a better understanding of the entire system. Lets improve the structure to three main sections:
1) Overview
2) Data Schemas (Rename terminologies)
3) Automated Generation - comprise of the installation and usage section
Users are able to manually generate the data schemas but this directory also contains helper scripts to automate the generation of these schemas. At the moment, such scripts are only applicable for the chemistry domain, and must be extended to work with other domains.